tests/experimental/datasources/test_spark_datasource.py

```diff
@@ -0,0 +1,275 @@
+import logging
+import pathlib
+from dataclasses import dataclass
+from typing import List
+
+import pytest
+
+import great_expectations.exceptions as ge_exceptions
+from great_expectations.alias_types import PathStr
+from great_expectations.data_context.util import file_relative_path
+from great_expectations.experimental.datasources.interfaces import (
+    BatchSortersDefinition,
+)
+from great_expectations.experimental.datasources.spark_datasource import (
+    CSVSparkAsset,
+    SparkDatasource,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def spark_datasource() -> SparkDatasource:
+    return SparkDatasource(name="spark_datasource")
```

Comment:
We should `pytest.skip` if we don't have Spark in our `test_backends`

Reply:
@cdkini Applied (committed).  Thanks!